TwitterTweetSentimentalAnalysis/credential_variables.env

```diff
@@ -0,0 +1,4 @@
+candidate_key =
```

Comment:
take this input from the user instead of creating this file, you can use `getpass` module to take password inputs which hides the text while entering. after that, you can also remove the `CredentialHelper.py` file and modify the `get_authenticated_api()` function also accordingly.

Reply:
Please make the changes as suggested